test/server/cards/02_SHD/units/CartelTurncoat.spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Cartel Turncoat', function() {
+    integration(function(contextRef) {
+        describe('Cartel Turncoat\'s Bounty ability', function() {
+            it('should draw a card', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['cartel-turncoat']
+                    },
+                    player2: {
+                        spaceArena: ['razor-crest#reliable-gunship']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.cartelTurncoat);
+                context.player1.clickCard(context.razorCrest);
+                expect(context.player2).toHaveExactPromptButtons(['Bounty: Draw a card', 'Pass']);
+                context.player2.clickPrompt('Bounty: Draw a card');
+                expect(context.player1.handSize).toBe(0);
+                expect(context.player2.handSize).toBe(1);
+                expect(context.player2).toBeActivePlayer();
+            });
+
+            it('should cause the opponent to take 3 damage to base if their deck is empty', function () {
```

Comment:
Don't need this test on any more of the draw bounties, I just did it once on the first one to confirm